### ISSUE TITLE
Remove brand icons?

### ISSUE THREAD

**Issue by locness3** [lucide-icons/lucide#94]
Since we don't add any brand icons, I think we should remove the current ones. What do you think?

**Comment by delnyn** [comment]
I agree

**Comment by ericfennis** [comment]
I agree, maybe we can move the current brand icons to a separate repo. I can image people want to use it

**Comment by ericfennis** [comment]
@locness3 @delnyn What about the idea, to create a separate package in packges folder called: 'lucide-brands'. We can move all the brand icons to this separate package, if people like to have this type of brand icons they can use the package.
In the readme we can notice that the brand icons are deprecated and we don't update/maintain them.

**Comment by delnyn** [comment]
Oh yes! Should I clone this one and remove the icons?

**Comment by locness3** [comment]
I don't know, but I think it'd be best if people didn't use them.

**Comment by mittalyashu** [comment]
How about create deprecated section for icons instead of completely removing them?

**Comment by dgdavid** [comment]
Just for the record, this [is/was also a concern in the Feather project](https://github.com/feathericons/feather/issues/763), where is stated that

> [simple-icons/simple-icons](https://github.com/simple-icons/simple-icons) repository already covers everything, uses SVG and the same 24x24 `viewBox`, so it could be a great replacement for people that need brand icons.

**Comment by mikeott** [comment]
Agreed. Brand icons have no business in icon sets.

**Comment by delnyn** [comment]
soooo? we just delete them all?

**Comment by ericfennis** [comment]
I'm thinking of defining the ultimate v1.0 for Lucide and I think this should be a part of it.

**Comment by delnyn** [comment]
what do you mean by "defining the ultimate v1.0 for Lucide"?

**Comment by SMAH1** [comment]
We are going with a lot of requests to add icons. I'm think split project. Part for medical, part for programing, part for brand , ... .
In this case, 'quality, version, update, ...' of each package are different.
In angular, we can one project for handle of use multi package icon library.

**Comment by delnyn** [comment]
I don't understand lol

**Comment by ericfennis** [comment]
> what do you mean by "defining the ultimate v1.0 for Lucide"?
@delnyn I will create a separate discussion for that later. I need write down some points for that.

**Comment by ericfennis** [comment]
@SMAH1 Hmm I'm not fan of this idea. I think it wil add some unnecessary complexity. And can be really confusion as well.

**Comment by karsa-mistmere** [comment]
Yeah, I agree, and apart from adding a layer unnecessary complexity, this would introduce the problem of having to deal with icons overlapping multiple categories. Categorising icons sounds like a potentially useful feature, but it's partly accomplished already via tagging. I would maybe add very basic filtering on tags to the lucide.dev UI and make sure that icon categories are properly filled in within icon tags.

**Comment by PodaruDragos** [comment]
Hello guys, 

If I can offer my 2 cents here, I do think it's fine to just remove them all-together.
There are other alternatives so it's fine, there's no need to overcomplicate things just for some existing icons.

**Comment by tryoxiss** [comment]
if I may chime in, I agree. They should be removed, maybe moved to a seperate `lucide-brands` or depreciated repository--where people would be able to add more brand icons, if they wanted. But they wouldn't be part of the main lucide. 

Especially since, due to lucides design guidelines, it's hard to accurately represent the logos while keeping them both recigniseable, good looking, and representative of the logo.

**Comment by delnyn** [comment]
yeah I think everyone agrees. this issue is literally older than my brother, please someone remove the brand icons

**Comment by locness3** [comment]
Interesting

**Comment by ericfennis** [comment]
I think this is something we should do for lucide 1.0 and create a separate package, like @tryoxiss mentions.

**Comment by danielbayley** [comment]
> From what I understand, these brand icons were removed, and the only way to get them now is to use [simple-icons](https://simpleicons.org/)?

@Sawtaytoes They haven’t actually been removed yet, but will be. In any case, I will be maintaining my own _lucide-brands_ fork retaining them and some more…

**Comment by tschoffelen** [comment]
I understand the logic for not adding any new brand icons, but I'd be against removing the current ones from the package. They fit in really well with the design language of the other icons, and make it easier for users to get started with just the single icon library.

**Comment by devlulcas** [comment]
> > From what I understand, these brand icons were removed, and the only way to get them now is to use [simple-icons](https://simpleicons.org/)?
> 
> @Sawtaytoes They haven’t actually been removed yet, but will be. In any case, I will be maintaining my own _lucide-brands_ fork retaining them and some more…

A lucide-brands package would be great. I want to keep using the same style for all the icons, but I need brand icons especially for social media stuff. If a new package emerges I will try to contribute.

**Comment by danielbayley** [comment]
@devlulcas They currently exist on the `brands` branch of my fork here: https://github.com/danielbayley/lucide/commits/brands. I haven’t decided exactly what to do with them in a more canonical way, yet… There are also a growing number of icons from rejected PRs here, which also should live somewhere. Possibly something like lucide-brands, and/or lucide-extra[s] packages… But need an easy way to hook into Lucide to add them to the fonts…

**Comment by FelipeJz** [comment]
I don't get it, these icons are useful, keep them 🥲

**Comment by tryoxiss** [comment]
> I don't get it, these icons are useful, keep them 🥲

My understanding is that there are 2 major issues with brand icons. Keep in mind, I am not a lawyer and this is just my understanding when I talk about the legalities, not legal advice.

1. **Design Guidelines & Trademarks**:  Only a small subset of brand icons could be legally added, as most brands dont allow you to modify thier logo (which would be required to meet lucides design guidelines). Additionally, many logos cannot be simplified to fit within lucides design guidelines even if it was legal to do so. There is [simpleicons.org](https://simpleicons.org/) which gets around this by not modifying the logo, and is a good alternative as thier icons are on the same sized canvas. (its also just better to have thier actual logo as its more recogniseable)

2. **Existing brand icons are shaky**: For the same reason as above legally. Many existing brand icons (to me at least) feel like they dont fit with the rest of the lucide icon kit.

Because new icons cant/wont be added, it feels weird to keep them around and creates a burden for the maintainers. People see brand icons, suggest new brand icons, maintainers need to reference the brand icon deprecation issue. If there were no brand icons, this would be less of a problem. And yes, the icon request template already makes people check off that they are not requesting a brand icon, and it still happens a lot)

**Comment by FelipeJz** [comment]
> > I don't get it, these icons are useful, keep them 🥲
> 
> My understanding is that there are 2 major issues with brand icons. Keep in mind, I am not a lawyer and this is just my understanding when I talk about the legalities, not legal advice.
> 
> 1. **Design Guidelines & Trademarks**:  Only a small subset of brand icons could be legally added, as most brands dont allow you to modify thier logo (which would be required to meet lucides design guidelines). Additionally, many logos cannot be simplified to fit within lucides design guidelines even if it was legal to do so. There is [simpleicons.org](https://simpleicons.org/) which gets around this by not modifying the logo, and is a good alternative as thier icons are on the same sized canvas. (its also just better to have thier actual logo as its more recogniseable)
> 2. **Existing brand icons are shaky**: For the same reason as above legally. Many existing brand icons (to me at least) feel like they dont fit with the rest of the lucide icon kit.
>    Because new icons cant/wont be added, it feels weird to keep them around and creates a burden for the maintainers. People see brand icons, suggest new brand icons, maintainers need to reference the brand icon deprecation issue. If there were no brand icons, this would be less of a problem. And yes, the icon request template already makes people check off that they are not requesting a brand icon, and it still happens a lot)

I'm not a lawyer either, correct me if i'm wrong, i think this is common sense:
1. Follow the guidelines, give the user the option to change the icon as they please so the burden of legality falls in the user not the library.
2. Ignore if it fits or not, this is a library and people do whatever they want with it anyway, make an exception for the brand logos and put it in a different category.
3. People will still ask for brand logos anyway, gash they are used everywhere, add them, leave it to the user to deal with the legal matters.

IMO this will force me, and others to look for libraries that contains these icons. Or add multiple libraries? Is this what we really want?

**Comment by tryoxiss** [comment]
> > > I don't get it, these icons are useful, keep them 🥲
> > 
> > 
> > My understanding is that there are 2 major issues with brand icons. Keep in mind, I am not a lawyer and this is just my understanding when I talk about the legalities, not legal advice.
> > 
> > 1. **Design Guidelines & Trademarks**:  Only a small subset of brand icons could be legally added, as most brands dont allow you to modify thier logo (which would be required to meet lucides design guidelines). Additionally, many logos cannot be simplified to fit within lucides design guidelines even if it was legal to do so. There is [simpleicons.org](https://simpleicons.org/) which gets around this by not modifying the logo, and is a good alternative as thier icons are on the same sized canvas. (its also just better to have thier actual logo as its more recogniseable)
> > 2. **Existing brand icons are shaky**: For the same reason as above legally. Many existing brand icons (to me at least) feel like they dont fit with the rest of the lucide icon kit.
> >    Because new icons cant/wont be added, it feels weird to keep them around and creates a burden for the maintainers. People see brand icons, suggest new brand icons, maintainers need to reference the brand icon deprecation issue. If there were no brand icons, this would be less of a problem. And yes, the icon request template already makes people check off that they are not requesting a brand icon, and it still happens a lot)
> 
> I'm not a lawyer either, correct me if i'm wrong, i think this is common sense:
> 
>     1. Follow the guidelines, give the user the option to change the icon as they please so the burden of legality falls in the user not the library.
> 
>     2. Ignore if it fits or not, this is a library and people do whatever they want with it anyway, make an exception for the brand logos and put it in a different category.
> 
>     3. People will still ask for brand logos anyway, gash they are used everywhere, add them, leave it to the user to deal with the legal matters.
> 
> 
> IMO this will force me, and others to look for libraries that contains these icons. Or add multiple libraries? Is this what we really want?

1. Providing the icon at all may make it fall on the library. It is often literally impossible to follow both Lucides design guidelines and the brand logo guidelines.
2. Saying "ignore if it fits" on a library almost entirely built around having icons that fit together is a really weak point.
3. Probably less, but even if not it takes effort to add them -- effort that nobody wants to do. If its so simple create your own library for brand icons following the Lucide guidelines.

Yes, but it is very common practice to use different libraries for most icons and brand icons, assuming you need brand icons at all (most projects dont). You dont even need to add another library for them, you can just copy+paste SVG code.

At any rate this issue is pretty settled, all the mainers seem to want to remove brand icons and its a decision the vast majority of the community seems to agree with. I was just trying to be helpful and give you the reasoning why to my understanding.

**Comment by FelipeJz** [comment]
> > > > I don't get it, these icons are useful, keep them 🥲
> > > 
> > > 
> > > My understanding is that there are 2 major issues with brand icons. Keep in mind, I am not a lawyer and this is just my understanding when I talk about the legalities, not legal advice.
> > > 
> > > 1. **Design Guidelines & Trademarks**:  Only a small subset of brand icons could be legally added, as most brands dont allow you to modify thier logo (which would be required to meet lucides design guidelines). Additionally, many logos cannot be simplified to fit within lucides design guidelines even if it was legal to do so. There is [simpleicons.org](https://simpleicons.org/) which gets around this by not modifying the logo, and is a good alternative as thier icons are on the same sized canvas. (its also just better to have thier actual logo as its more recogniseable)
> > > 2. **Existing brand icons are shaky**: For the same reason as above legally. Many existing brand icons (to me at least) feel like they dont fit with the rest of the lucide icon kit.
> > >    Because new icons cant/wont be added, it feels weird to keep them around and creates a burden for the maintainers. People see brand icons, suggest new brand icons, maintainers need to reference the brand icon deprecation issue. If there were no brand icons, this would be less of a problem. And yes, the icon request template already makes people check off that they are not requesting a brand icon, and it still happens a lot)
> > 
> > I'm not a lawyer either, correct me if i'm wrong, i think this is common sense:
> > 
> >     1. Follow the guidelines, give the user the option to change the icon as they please so the burden of legality falls in the user not the library.
> > 
> >     2. Ignore if it fits or not, this is a library and people do whatever they want with it anyway, make an exception for the brand logos and put it in a different category.
> > 
> >     3. People will still ask for brand logos anyway, gash they are used everywhere, add them, leave it to the user to deal with the legal matters.
> > 
> > 
> > IMO this will force me, and others to look for libraries that contains these icons. Or add multiple libraries? Is this what we really want?
> 
> 1. Providing the icon at all may make it fall on the library in addition to the user. It is often literally impossible to follow both Lucides design guidelines and the brand logo guidelines, so its better to just not.
> 2. Saying "ignore if it fits" on a library almost entirely built around having icons that fit together is a really weak point. And if your making an exception for brand logos, just use a different library or get the brand icons directly from the source. If not in lucides style, why should lucide add them at all?
> 3. Probably less, but even if not it takes effort to add them -- effort that nobody wants to do. If its so simple create your own library for brand icons following the Lucide guidelines.
> 
> Yes, but it is very common practice to use different libraries for most icons and brand icons, assuming you need brand icons at all (most projects dont). You dont even need to add another library for them, you can just copy+paste SVG code.
> 
> At any rate this issue is pretty settled, all the mainers seem to want to remove brand icons and its a decision the vast majority of the community seems to agree with. I was just trying to be helpful and give you the reasoning why to my understanding.

I get it, I do appreciate your insight in this matter, I'm just a little bit frustrated because this library has been my go to solution for everything, but now I'll have to depend on another library just for brand logos which is sad. But as a developer, I get it. Thanks

**Comment by navidmadannezhad** [comment]
To be honest I don't think this is going to be a good idea. 
Icon libraries play an important role in providing a "single source of icons" in a project, the developer shouldn't get confused about which libraries to install for a specific set of icons, or use some icons from a certain library and user others as a custom svg component.

There are many projects out there which need to use brand icons, you guys can literally have important brand icons on your library and remove the rest that make problem, or even better, as someone said in previous comments, put all brand icons in a "lucide-brands" library.

I don't think it would be much of a correct solution to remove important brand icons like Linkedin and Instagram (which many websites use them) because some rare brand icon might cause some law-related issues.

**Comment by DesignSkeptic** [comment]
They are already marked as deprecated. Just remove them.

**Comment by farzadasgari** [comment]
I respect the reasoning behind keeping the icon set neutral by avoiding brand icons. However, I believe removing existing brand icons might be a step backward for developers who rely on them for consistent styling in projects (e.g., social media integrations). Brand icons are a common need, and their removal could push users to mix Lucide with other libraries. I believe removing widely used ones like GitHub and LinkedIn was a misstep.

**Comment by un33k** [comment]
Just create a /contrib folder or a new repo and move them there. Instead of crippling this icon package, enrich it with more icons. Moonwalk doesn’t belong here. People don’t install ten icon packages, if they can’t find a brand icon, they move on.
